src/lib/journey/journey.store.ts

```diff
@@ -54,10 +54,25 @@ export function initialize(initOptions?: StepOptions): JourneyStore {
     });
 
     try {
-      /**
-       * Initial attempt to retrieve next step
-       */
-      nextStep = await FRAuth.next(prevStep as FRStep, options);
+      if (resumeUrl) {
+        // If resuming an unknown journey remove the tree from the options
+        options.tree = undefined;
```

Comment:
If `tree` exists, the SDK will append `authIndexType` and `authIndexValue` to the `/authenticate` call. If this value differs from the tree associated with the `suspendedId`, it will not work.

Reply:
Actually having tested this change and observed the network requests, it seems that suspendedId is all that the authenticate endpoint needs to resume, so everything you said above makes sense 👍